### PR TITLE
save violations in html file

### PR DESCRIPTION
**Pull request by BeataKr**

### CODE REVIEW

**Review by abhinaba-ghosh — CHANGES_REQUESTED** [review]
Thanks for the PR. 
Can you accommodate the HTML publication inside the `checkA11y` method itself? It will help the user use the same interface just by providing the HTML options param. Better to expose an HTML specific reporter class [here](https://github.com/abhinaba-ghosh/axe-playwright/tree/master/src/reporter) and pass the same in `checkA11y` method.

**Comment by BeataKr** [comment]
Yes, I'll try to reach your suggestion in this PR.

File: package.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
     "typescript": "^4.1.3"
   },
   "dependencies": {
-    "axe-core": "^4.0.1"
+    "axe-core": "^4.0.1",
```
Comment (BeataKr) [inline package.json:40]: also, I think that package should be raised to 4.2.*

Comment (abhinaba-ghosh) [reply package.json:40]: feel free to update it.

**Comment by abhinaba-ghosh** [comment]
hey @BeataKr any update on this?

**Comment by BeataKr** [comment]
Hi @abhinaba-ghosh  sorry didn't have time, I'll back to this soon

**Comment by abhinaba-ghosh** [comment]
Hi @BeataKr , are you considering updating the PR. It seems helpful for others.

**Comment by abhinaba-ghosh** [comment]
Hi @BeataKr , calling here again. Are you looking for handle the PR comments and merge the PR. Let me know how I can help.

**Comment by ankit0305** [comment]
@abhinaba-ghosh Is the save to html feature integrated or is in pending state?

**Comment by abhinaba-ghosh** [comment]
Hey there! At this time it will be difficult for me to fully concentrate here considering organization priorities. I will happily collaborate if anyone wants to tale it up on priority.

**Comment by manishmitraqa** [comment]
Quick question : @BeataKr  - It will be great , if this pr is merged, I see The checks have passed as well

**Review by manishmitraqa — APPROVED** [review]

**Comment by ilolzzz** [comment]
hey, any chance this can be merged soon? 
thank you

**Comment by jmh-bjss** [comment]
Hi all, are there any outstanding blockers or can this be merged?

**Comment by BeataKr** [comment]
I had too many conflicts, created new one https://github.com/abhinaba-ghosh/axe-playwright/pull/209

**Pull request closed without merging**